### ISSUE TITLE
after updating to 8.3 I am still seeing node read errors

### ISSUE THREAD

**Issue by bietiekay** [altmp/altv-issues#1273]
### Description of the problem

Starting the server and connecting one user yields to error messages and server getting unresponsive.

```
[11:15:50][Error] Vehicle [ID: 1071] node read: Automobile_Sector(14), node size = 26, readed = 23 type: 13
[11:15:50][Error] Vehicle [ID: 1071] node read: Automobile_SectorPosition(15), node size = 36, readed = 26 type: 13
[11:15:50][Error] Vehicle [ID: 1071] node read: Automobile_EntityOrientation(16), node size = 35, readed = 24 type: 13
[11:15:50][Error] Vehicle [ID: 1071] node read: Automobile_PhysicalVelocity(17), node size = 36, readed = 35 type: 13
[11:15:50][Error] Vehicle [ID: 1071] node read: Automobile_VehicleAngVelocity(18), node size = 2, readed = 2 type: 13
[11:15:50][Error] Vehicle [ID: 1071] node read: Automobile_VehicleSteering(19), node size = 10, readed = 4 type: 13
[11:15:50][Error] Vehicle [ID: 1071] node read: Automobile_VehicleControl(20), node size = 22, readed = 10 type: 13
[11:15:50][Error] Vehicle [ID: 1071] node read: Automobile_VehicleGadget(21), node size = 3, readed = 3 type: 13
[11:15:50][Error] Vehicle [ID: 1071] node read: Automobile_VehicleTask(13), node size = 18, readed = 18 type: 13
[11:15:50][Error] Vehicle [ID: 2294] node read: Automobile_Sector(14), node size = 26, readed = 23 type: 13
[11:15:50][Error] Vehicle [ID: 2294] node read: Automobile_SectorPosition(15), node size = 36, readed = 26 type: 13
[11:15:50][Error] Vehicle [ID: 2294] node read: Automobile_EntityOrientation(16), node size = 35, readed = 24 type: 13
[11:15:50][Error] Vehicle [ID: 2294] node read: Automobile_PhysicalVelocity(17), node size = 36, readed = 35 type: 13
..
...
...
```

### Reproduction steps

Start server and spawn vehicles.

### Expected behaviour

No error - 

### Operating system

Windows 10

### Version

8.3

**Comment by FabianTerhorst** [comment]
Please provide reproduction code.

**Comment by bietiekay** [comment]
additional LOG in console: 
```
[11:27:39][Error] class CVehicleComponentReservationDataNode
[11:27:39][Error] Veh node read failed 11 6 41 000002AA0B769308
[11:27:39][Error] class CVehicleTaskDataNode
[11:27:39][Error] Veh node read failed 13 143 18 000002AA0B769410
[11:27:39][Error] class CSectorPositionDataNode
[11:27:39][Error] Veh node read failed 15 36 26 000002AA0B7677B8
[11:27:39][Error] class CEntityOrientationDataNode
[11:27:39][Error] Veh node read failed 16 35 36 000002AA0B767F30
[11:27:39][Error] class CPhysicalVelocityDataNode
[11:27:39][Error] Veh node read failed 17 36 35 000002AA0B768030
[11:27:39][Error] class CEntityScriptInfoDataNode
[11:27:39][Error] Veh node read failed 7 135 129 000002AA0B767A68
[11:27:39][Error] class CVehicleDamageStatusDataNode
[11:27:39][Error] Veh node read failed 10 7 194 000002AA0B768DC0
[11:27:39][Error] class CVehicleComponentReservationDataNode
[11:27:39][Error] Veh node read failed 11 6 41 000002AA0B769308
[11:27:39][Error] class CVehicleTaskDataNode
[11:27:39][Error] Veh node read failed 13 143 18 000002AA0B769410
[11:27:39][Error] class CSectorPositionDataNode
[11:27:39][Error] Veh node read failed 15 36 26 000002AA0B7677B8
[11:27:39][Error] class CEntityOrientationDataNode
[11:27:39][Error] Veh node read failed 16 35 36 000002AA0B767F30
[11:27:39][Error] class CPhysicalVelocityDataNode
[11:27:39][Error] Veh node read failed 17 36 35 000002AA0B768030
[11:27:39][Error] class CEntityScriptInfoDataNode
[11:27:39][Error] Veh node read failed 7 135 129 000002AA0B767A68
[11:27:39][Error] class CVehicleDamageStatusDataNode
[11:27:39][Error] Veh node read failed 10 7 194 000002AA0B768DC0
[11:27:39][Error] class CVehicleComponentReservationDataNode
[11:27:39][Error] Veh node read failed 11 6 41 000002AA0B769308
[11:27:39][Error] class CVehicleTaskDataNode
[11:27:39][Error] Veh node read failed 13 15 37 000002AA0B769410
[11:27:39][Error] class CSectorPositionDataNode
[11:27:39][Error] Veh node read failed 15 36 26 000002AA0B7677B8
[11:27:39][Error] class CEntityOrientationDataNode
[11:27:39][Error] Veh node read failed 16 35 36 000002AA0B767F30
[11:27:39][Error] class CPhysicalVelocityDataNode
[11:27:39][Error] Veh node read failed 17 36 35 000002AA0B768030
[11:27:39][Error] class CEntityScriptInfoDataNode
[11:27:39][Error] Veh node read failed 7 135 129 000002AA0B767A68
[11:27:39][Error] class CVehicleDamageStatusDataNode
[11:27:39][Error] Veh node read failed 10 7 194 000002AA0B768DC0
[11:27:39][Error] class CVehicleComponentReservationDataNode
[11:27:39][Error] Veh node read failed 11 6 41 000002AA0B769308
[11:27:39][Error] class CVehicleTaskDataNode
[11:27:39][Error] Veh node read failed 13 143 18 000002AA0B769410
[11:27:39][Error] class CSectorPositionDataNode
[11:27:39][Error] Veh node read failed 15 36 26 000002AA0B7677B8
[11:27:39][Error] class CEntityOrientationDataNode
[11:27:39][Error] Veh node read failed 16 35 36 000002AA0B767F30
[11:27:39][Error] class CPhysicalVelocityDataNode
[11:27:39][Error] Veh node read failed 17 36 35 000002AA0B768030
```

**Comment by bietiekay** [comment]
It happens on connect - when there are vehicles streamed into the client. When I drive around I get additional node read errors with vehicles that get spawned in.

**Comment by Tsumiki-Chan** [comment]
Seems to be fixed with 8.4

**Comment by bietiekay** [comment]
yes seems fixed with new client version (server still on 8.3 -> no error messages, client got apparently updated)
